src/renderer/Framebuffer.cpp

```diff
@@ -113,6 +113,13 @@ void CFramebuffer::release() {
     m_pStencilTex   = nullptr;
 }
 
+void CFramebuffer::abandon() {
```

Comment:
this should be called "leak" and at that point you start asking: why is this here? Can't a shared_ptr fix it?